.travis.yml

```diff
@@ -48,6 +50,8 @@ jobs:
       scala: 2.11.12
     - <<: *test
       scala: 2.11.12
+    - <<: *microsite
```

Comment:
We won't want a microsite for 2.11. It will look identical to 2.12 and overwrite the 2.12 one.